front/src/modules/activities/components/TaskGroups.tsx

```diff
@@ -1,9 +1,62 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { IconCheckbox } from '@tabler/icons-react';
+
+import { Button, ButtonVariant } from '@/ui/button/components/Button';
+import { ActivityType } from '~/generated/graphql';
+
+import { useOpenCreateActivityDrawer } from '../hooks/useOpenCreateActivityDrawer';
 import { useTasks } from '../hooks/useTasks';
 
 import { TaskList } from './TaskList';
 
+const StyledTaskGroupEmptyContainer = styled.div`
+  align-items: center;
+  align-self: stretch;
+  display: flex;
+  flex: 1 0 0;
+  flex-direction: column;
+  gap: 8px;
+  justify-content: center;
+  padding: 12px 16px 64px 16px;
```

Comment:
We should probably use spacing(x) function here

Reply:
Good point!

Reply:
Good point @FelixMalfait!